CODEOWNERS

```diff
@@ -259,6 +259,7 @@
 /scripts/sanity_chk/expr_parser.py        @andrewboie @nashif
 /scripts/gen_app_partitions.py            @andrewboie
 /scripts/partition_manager.py             @SebastianBoe
+/scripts/partition_manager_report.py      @SebastianBoe
```

Comment:
You can change these to be you or me I guess. @SebastianBoe  was involved in the build code, but not the scripts themself. I am not sure if downstream code owners can be listed here though?

Reply:
Fixed